CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+Contributing to Project TrainSite Front End
+===========================================
+
+Yay! Thanks for considering to contribute to this project!
+To make sure your contribution can be merged into the main repository, follow this short guide.
+
+
+## Workflow
+
+- **Fork this repository** using the "Fork" button in the top right corner
+- **Create a feature branch** based off of the most recent commit on `develop` following this naming convention: `feature/<short-description>` (see more [below](#git-flow))
+- **Do your thing!** 👩‍💻 👨‍💻
+- **Write tests** for everything you added/modified to make sure everything works as expected
+- **Make a pull request** to this repository's `develop` branch
+- **Be happy!** 🎉
+
+
+## Guidelines
+
+Please follow these guidelines Whilst developing. This helps keeping the code nice and organized.
+
+- Don't work directly on the `master` or `develop` branches (see [below](#git-flow))!
+- Run `npm run format && npm run lint` every time before creating a new commit!
+- If you encounter formatting issues, try `npm run format`
+- If you encounter formatting/linting issues, try `npm run fix`
+- Run `npm test` every time before submitting a pull request
+
+*Note: Formatting and linting will be done through git hooks automatically for you!*
```

Comment:
Looks like we'll need to tweak this section a bit as we don't have all these commands here...

Reply:
Yep, should've done that first thing, was dead tired (=lazy) last night.   
I tweaked it now to fit our repo as it is for now. Have to remember to update this when we set  up the develop-branch or get some new scripts going.